ledger-core/runner/bench_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package runner
+
+import (
+	"testing"
+
+	"github.com/insolar/assured-ledger/ledger-core/application/builtin/contract/testwallet"
+	testwalletProxy "github.com/insolar/assured-ledger/ledger-core/application/builtin/proxy/testwallet"
+	"github.com/insolar/assured-ledger/ledger-core/insolar"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/instrumentation/inslogger/instestlogger"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/runner/execution"
+	"github.com/insolar/assured-ledger/ledger-core/testutils/gen"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/descriptor"
+)
+
+func BenchmarkRunnerService(b *testing.B) {
+	runner := NewService()
+	_ = runner.Init()
+
+	var (
+		ctx = instestlogger.TestContext(b)
+
+		object        = gen.UniqueGlobalRef()
+		remoteObject  = gen.UniqueGlobalRef()
+		class         = testwalletProxy.GetClass()
+		wallet, _     = testwallet.New()
+		defaultObject = insolar.MustSerialize(wallet)
+	)
+
+	executionContext := execution.Context{
+		ObjectDescriptor: descriptor.NewObject(object, reference.Local{}, class, defaultObject),
+		Context:          ctx,
+		Request: &payload.VCallRequest{
+			CallType:       payload.CTMethod,
+			CallSiteMethod: "GetBalance",
+			Arguments:      insolar.MustSerialize([]interface{}{remoteObject, uint32(100)}),
+		},
+		Sequence: 0,
+		Object:   object,
+	}
+
+	b.ResetTimer()
+	b.ReportAllocs()
+
+	c := make(chan struct{})
+
+	for i := 0; i < b.N; i++ {
```

Comment:
You can, also, add cheap benchmark for parallel calls

Reply:
runStart spawns goroutine. So I don't see any point of parallel calls. And it is outside of ticket scope.

Reply:
Just move `Runner` preparation and `executionContext` to separate function, also use `b.Run()` to remove duplication of code

Reply:
I don't understand. Is this benchmark wrong? What does parallel benchmark will prove/bench?